discovery-provider/src/tasks/entity_manager/entity_manager.py

```diff
@@ -157,6 +168,19 @@ def entity_manager_update(
         session.bulk_save_objects(records_to_save)
         num_total_changes += len(records_to_save)
 
+        # update metrics
+        metric_latency.save_time()
+        metric_num_changed.save(
+            len(new_records["playlists"]), {"entity_type": EntityType.PLAYLIST.value}
+        )
+        metric_num_changed.save(
+            len(new_records["tracks"]), {"entity_type": EntityType.TRACK.value}
+        )
+        metric_num_changed.save(num_total_changes, {"entity_type": "all"})
```

Comment:
Let's remove this line and compute "all" using any of the following PromQL queries on the Grafana side:

```
entity_manager_update_num_changed{...}
entity_manager_update_num_changed{..., entity_type~=".*"}
entity_manager_update_num_changed{..., entity_type~="track"} + entity_manager_update_num_changed{..., entity_type~="playlist"}
sum without(entity_type) entity_manager_update_num_changed{...}
```

Doing so removes the ~11 auto-generated metrics (sum, count, buckets) from being generated and tracked by the exporter and prevents these "duplicate" metrics from being stored on Prometheus.